### PR TITLE
[MGDSTRM-9626] Expose KFM Admin API SSO variables, default to MAS-SSO

### PR DESCRIPTION
**Pull request by MikeEdgar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -249,6 +249,18 @@ the variable may be set to `redhat_sso` and additional configuration can be prov
 `REDHAT_SSO_CLIENT_ID` (required), and `REDHAT_SSO_CLIENT_SECRET` (required). See the description for each variable in the [kas-installer-defaults.env](kas-installer-defaults.env)
 file for more information.
 
+### kas-fleet-manager admin API SSO
+
+The kas-fleet-manager will use MAS-SSO as an identity provider for its admin API endpoints by default. An alternate IdP
+may be provided by setting `ADMIN_API_SSO_BASE_URL`, `ADMIN_API_SSO_REALM`, and `ADMIN_API_SSO_ENDPOINT_URI` in your
+`kas-installer.env` configuration, or in the environment. See `kas-installer-defaults.env` for the default values.
```
Comment (machi1990) [inline README.md:256]: let's also mention the `ADMIN_AUTHZ_CONFIG` which allows to set the roles if someone uses a different IDP: https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/9d571b351508cec7c488a7da3410b1c312a73eba/templates/service-template.yml#L175

**Review by machi1990 — CHANGES_REQUESTED** [review]
left a small change request, otherwise lgtm. Thanks

File: README.md (hunk 1 above)
Comment (MikeEdgar) [reply README.md:256]: Good idea, thank you - I just added a some guidance on how to configure it.

**Review by machi1990 — APPROVED** [review]
thank you @MikeEdgar

**Pull request merged**